### ISSUE TITLE
feature: Compatibility with New Tidy 5e Sheets

### ISSUE THREAD

**Issue by kgar** [cstby/foundryvtt-world-currency-5e#27]
Hello, I am the author of the new Tidy 5e Sheets modules. The new sheets are structured fairly differently than the default/legacy sheets, and they operate on a different render cycle (more aligned with Application V2, coming in Foundry V12), because they are rendered with the svelte framework.

I would like to establish compatibility with this module in a way that will work for the long haul. I keep an [API](<https://kgar.github.io/foundry-vtt-tidy-5e-sheets/>) for integrating into Tidy. I do not yet have API functionality in place yet for overriding things related to currency, but I can absolutely set that up. I just need an idea of what things you'd like to override and what things you'd like to hide, etc.

Also, I am glad to submit a PR with wire-up code, if that is preferred.

I hope to work together, and I know our mutual users would benefit a lot from it. Let me know your thoughts.

**Comment by KyaWither** [comment]
Using Foundry V11 315 and D&D 5e 2.4.1 59

Everything works fine with World Currency and Tidy 5e Sheets, except the hiding of currency. It does not reflect on Tidy, but it does on the legacy sheets. 

So trying to hide a currency like EP does not reflect at all on Tidy. Keep up the good work everyone <3 thank you

**Comment by cstby** [comment]
@kgar Thanks for reaching out! I've been using Tidy 5e for over three years now, and I'm happy to see that you're planning on continuing development after the release of DND5E v3.

As @kmuld mentioned above, our latest v2.0.3 release should fix compatibility with Tidy 5e Sheets. It's all working as expected on my local setup, but definitely let us know if you're seeing something different on your end.

Regarding the API, I definitely share your goal of establishing compatibility that will work in the long haul. We have a few internal decisions to make before we can provide you API requirements. Once that's settled, we'll loop back to you with a proposal. If you're planning to make any currency changes to Tidy 5e (e.g. adding icons like the v3 sheet), that would be helpful context for us to have at this point. No worries at all though if you don't have any currency changes on your roadmap.

**Comment by KyaWither** [comment]
> > Using Foundry V11 315 and D&D 5e 2.4.1 59
> > Everything works fine with World Currency and Tidy 5e Sheets, except the hiding of currency. It does not reflect on Tidy, but it does on the legacy sheets.
> > So trying to hide a currency like EP does not reflect at all on Tidy. Keep up the good work everyone <3 thank you
> 
> Which version of World Currency are you using? This should be fixed in 2.0.3.

I believe the problem might be because I'm using D&D 5e 2.4.1. I can't use the latest Tiny because I'm not on D&D 3.0, which I'm avoiding like many others for now. I've been using the latest version of World Currency though.

**Comment by cstby** [comment]
@KyaWither Can you open a separate issue and note your Tidy 5e version as well? I'm converting this issue into a discussion of future compatibility. We'll take a look and see whether we can support older versions of Tidy. No guarantee, but at the very least we can document the oldest version of Tidy 5e that's still compatible.
